libmobilecoin/src/transaction.rs

```diff
@@ -541,6 +546,13 @@ pub extern "C" fn mc_transaction_builder_add_change_output(
             .as_slice_mut_of_len(TxOutConfirmationNumber::size())
             .expect("out_tx_out_confirmation_number length is insufficient");
 
+        // TODO: If you want to support mixed transactions, use something other
```

Comment:
Refer to https://github.com/mobilecoinfoundation/mobilecoin/pull/1827/files#r859868566